app/views/exercises/implement.html.slim

```diff
@@ -36,3 +47,4 @@
 
 
   = render('editor', exercise: @exercise, files: @files, submission: @submission)
+  = render('shared/modal', classes: 'modal-lg', id: 'modal-info-pair-programming', template: 'application/info_pair_programming', title: "Pair Programming Info")
```

Comment:
Potentially, you could / should use the translation `shared.pair_programming_info`:

```suggestion
  = render('shared/modal', classes: 'modal-lg', id: 'modal-info-pair-programming', template: 'application/info_pair_programming', title: t('shared.pair_programming_info'))
```